src/resources/postcss/dialog.pcss

```diff
@@ -0,0 +1,138 @@
+/* = Dialogs
+This can include modals, toasters, confirms. alerts.
+============================ */
+
+/* vars */
+.tribe-dialog {
+	--tribe-dialog-background-color: #FFF;
+	--tribe-dialog-close-background: #FFF;
+	--tribe-dialog-close-border-color: #BABABA;
+	--tribe-dialog-close-border-width: 1px;
+	--tribe-dialog-close-bottom: auto;
+	--tribe-dialog-close-color: #BABABA;
+	--tribe-dialog-close-height: 1.5em;
+	--tribe-dialog-close-left: auto;
+	--tribe-dialog-overlay-color: transparent;
+	--tribe-modal-overlay-color: rgba(20,24,39,0.9);
+	--tribe-dialog-border-radius: 4px;
+}
+
+/* The dialog/popup/modal itself */
+.tribe-dialog {
+	height: 100vh;
+	position: fixed;
+	width: 100vw;
+	top: 0;
+	left: 0;
+
+	&[aria-hidden='true'] {
+		display: none;
+	}
+}
+
+/* The trigger button" */
+.tribe-dialog__trigger {}
+
+/* The overlay */
+.tribe-dialog__overlay {
+	background-color: var(--tribe-dialog-overlay-color);
+	height: 100vh;
+	left: 0;
+	opacity: .9;
+	position: fixed;
+	top: 0;
+	width: 100vw;
+}
+
+/* Content wrapper - includes close button*/
+.tribe-dialog__wrapper {
+	background-color: var(--tribe-dialog-background-color);
+	border-radius: var(--tribe-dialog-border-radius);
+	box-shadow: 0 2px 54px 0 rgba(0,0,0,0.1);
+	left: 50%;
+	max-height: 100vh;
+	min-width: 50%;
+	overflow-y: scroll;
+	padding: 16px 28px;
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%) translateX(-50%);
+
+	div[role="document"] {
+		align-items: flex-end;
+		display: flex;
+		flex-flow: column;
+		justify-content: space-between;
+	}
+}
+
+/* The "close" button */
+.tribe-dialog__close-button {
+	background: var(--tribe-dialog-close-background);
+	cursor: pointer;
+	display: inline-block;
+	font-size: 14px;
+	height: var(--tribe-dialog-close-height);
+	line-height: var(--tribe-dialog-close-height);
+	padding: 0;
+	position: absolute;
+	top: calc( .25em + var(--tribe-dialog-close-height) );
+	width: var(--tribe-dialog-close-height);
+	z-index: 1;
+
+	&--hidden {
```

Comment:
Personal preference, but for these modifier classes, I prefer to type it all out (in this case, `.tribe-dialog__close-button--hidden`, so that if we're searching for it and want to modify the styles, we can find it. Doing it this way makes maintenance more difficult.

Reply:
Sorry, old habits die hard - will correct.